todolist/app/components/Checkbox/layout.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Text, StyleSheet, TouchableOpacity, CheckBox } from 'react-native';
+
+const styles = StyleSheet.create({
```

Comment:
extract this to a new file, at the same level of layout.js called 'styles.js'. The import the styles here..